roles/zuul/templates/etc/zuul/config/layout.yaml

```diff
@@ -74,12 +74,11 @@ projects:
 
   - name: BonnyCI/sandbox
     check_github:
-      - echo-true
-      - noop
+      - 'bonnyci-run-{pipeline}':
```

Comment:
We also need to create a job from the job-template. We don't currently have any job-templates, so we will need to create `roles/zuul/files/jobs/projects.yaml` similar to [infra's projects.yaml](http://git.openstack.org/cgit/openstack-infra/project-config/tree/jenkins/jobs/projects.yaml) with the job-template and parameters in the jobs section of a BonnyCI/sandbox project.